### PR TITLE
improvement(hydra): Update argus to 0.11.3 (siren-stable)

### PR DESCRIPTION
**Pull request by k0machi**
This update adjust sirenada client behaviour to not change error status
into aborted inside test cases.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by k0machi** [comment]
> @ please, fix merge conflicts, https://github.com/scylladb/scylla-cluster-tests/pull/6158/files#diff-bcc5c95132ad356dadca302f1ef433e3c13194d2b0818350134a2ac6a2389089 is merged

This is a PR for `siren-stable` branch, there shouldn't be a conflict here

**Review by enaydanov — APPROVED** [review]

**Comment by fruch** [comment]
@k0machi  is #6196 enough ?

**Comment by k0machi** [comment]
> @k0machi is #6196 enough ?

if it will be backported to siren-stable branch then yes, otherwise this is still needed for sirenada runs.

File: requirements.in [hunk 1]
```diff
@@ -48,6 +48,7 @@ azure-mgmt-resource==20.0.0
 azure-mgmt-subscription==1.0.0
 azure-mgmt-resourcegraph==8.0.0
 pydantic==1.8.2
+typing-extensions==4.6.2 # Until pydantic is updated >=1.10.4
```
Comment (temichus) [inline requirements.in:51]: typing-extensions==4.5.0 ?

**Review by temichus — APPROVED** [review]

**Comment by fruch** [comment]
@k0machi please check the target branch, I think it should be update regularly from master, and no needs to have updates specifically to it

**Comment by k0machi** [comment]
> @k0machi please check the target branch, I think it should be update regularly from master, and no needs to have updates specifically to it

It hasn't been updated since March, I'll ask Evgeniy when he will reset + ff this branch

**Comment by enaydanov** [comment]
Unfortunately, our automatic siren-stable branch update is broken right now (mostly because of broken envs). We do manual update of this branch.

**Pull request merged**